README.md

```diff
@@ -57,16 +73,11 @@ Incubator Charts are those that do not meet these criteria. Having the incubator
 
 In order to get a Chart from incubator to stable, Chart maintainers should open a pull request that moves the chart folder.
 
-## Supported Kubernetes Versions
-
-This chart repository supports the latest and previous minor versions of Kubernetes. For example, if the latest minor release of Kubernetes is 1.8 then 1.7 and 1.8 are supported. Charts may still work on previous versions of Kubernertes even though they are outside the target supported window.
 
 To provide that support the API versions of objects should be those that work for both the latest minor release and the previous one.
```

Comment:
this is related to Kubernetes, not the repo structure. It is now out of place

Reply:
Thanks for finding this. This was a cut and paste error (left-behind text). I moved this text to the K8S version support statement.